card/src/main/java/com/airwallex/android/card/CardComponent.kt

```diff
@@ -124,6 +127,15 @@ class CardComponent : ActionComponent {
                 }
             }
             return true
+        } else if (requestCode == ThreeDSecurityActivityLaunch.REQUEST_CODE) {
+            val result = ThreeDSecurityActivityLaunch.Result.fromIntent(data)
+            result?.paymentIntentId?.let {
+                listener?.onCompleted(AirwallexPaymentStatus.Success(it))
+            }
+            result?.exception?.let {
+                listener?.onCompleted(AirwallexPaymentStatus.Failure(it))
+            }
```

Comment:
Is there a possibility that the listener never completes if both of the above params are nil